angular/projects/admin/src/app/admin/settings/settings-dialog-manage-languages/settings-dialog-manage-languages.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="row">
```

Comment:
Add a title to the modal and using `<mat-dialog-title>`. See: https://material.angular.io/components/dialog/overview#dialog-content

Reply:
Solved on this commit https://github.com/oddbit/tanam/pull/51/commits/0f799bcd17b0da668755699d7cc322d045e11e05